custom-metrics-stackdriver-adapter/examples/direct-to-sd/README.md

```diff
@@ -114,9 +118,9 @@ This should return a MetricValueList object. If you get a 404, the metric timese
 custom-metrics-stackdriver-adapter.
 
 Make sure that:
-* your metric is correctly labeled with pod id. 
+* your metric is correctly labeled with pod details. 
```

Comment:
pod details (id, name and namespace), details alone is too vague.

Reply:
Done